bigtable/use-cases/fraudDetection/README.md

```diff
@@ -116,22 +116,82 @@ terraform apply -var="project_id=$PROJECT_ID"
+|-------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| Cloud Bigtable Instance             | featurestore-{RANDOM\_ID}                                                                                                                   |
+| Cloud Bigtable Table                | customer-information-{RANDOM\_ID}                                                                                                           |
+| Cloud Bigtable Column Family        | demographics, history                                                                                                                       |
+| Cloud Pubsub Input Topic            | transaction-stream-{RANDOM\_ID}                                                                                                             |
+| Cloud Pubsub Output Topic           | fraud-result-stream-{RANDOM\_ID}                                                                                                            |
+| Cloud Pubsub Output Subscription    | fraud-result-stream-subscription-{RANDOM\_ID}                                                                                               |
+| Google Storage Bucket               | fraud-detection-{RANDOM\_ID}                                                                                                                |
+| Google Storage Objects              | * temp/ (*for temporary dataflow generated files*) <br/> * testing_dataset/ <br/>* training_dataset/ <br/>* ml_model/                       |
+| VertexAI Model                      | fraud-ml-model-{RANDOM\_ID}                                                                                                                 |
+| VertexAI Endpoint                   | *The endpoint Id is determined in runtime, stored in Scripts/ENDPOINT\_ID.output*                                                           |
+| Dataflow Load Demographics Data Job | load-customer-demographics-{RANDOM\_ID} (*batch job that loads demographics data from GS to Cloud Bigtable*)                                |
+| Dataflow Load Historical Data Job   | load-customer-historical-transactions-{RANDOM\_ID} (*batch job that loads historical data from GS to Cloud Bigtable*)                       |
+| Dataflow Fraud Detection Job        | fraud-detection-{RANDOM\_ID} (*streaming job that listens to the input Pub/Sub topic and produces the results to the output Pub/Sub topic*) |
+
+*Note: For simplicity, running the Terraform commands will build the
+infrastructure mentioned above and then run 2 Java programs that will create
+the Dataflow pipelines. Alternatively, you could create a Dataflow template and
+let Terraform deploy those templates rather than running the Java code inside
+Terraform code.*
```

Comment:
I remember I was talking to Alexey and he said that he doesn't like the fact that we are running java code inside Terraform. Therefore, we wanted to highlight that alternatively, people could create a Dataflow template and run it from inside terraform rather than launching the pipeline by running Java code locally.